core/web/job_runs_controller.go

```diff
@@ -54,6 +54,7 @@ func (jrc *JobRunsController) Index(c *gin.Context, size, page, offset int) {
 // Create starts a new Run for the requested JobSpec.
 // Example:
 //  "<application>/specs/:SpecID/runs"
+// TODO: move this to v2
 func (jrc *JobRunsController) Create(c *gin.Context) {
```

Comment:
This refers to a v1 JobRun. In V2 we use the `JobsController` and the `PipelineRunsController`

Reply:
roger, moved to `jobs_controller` and added the `CronJob` case there + validation